start-a-local-cluster-in-docker.md

```diff
@@ -53,7 +53,7 @@ We've used `roachnet` as the network name here and in subsequent steps, but feel
 --net<span class="o">=</span>roachnet <span class="sb">`</span>
 -p 26257:26257 -p 8080:8080 <span class="sb">`</span>
 -v <span class="s2">"//c/Users/&lt;username&gt;/cockroach-data/roach1:/cockroach/cockroach-data"</span> <span class="sb">`</span>
-cockroachdb/cockroach:beta-20170209 <span class="nb">start</span> --insecure</code></pre></div>
+cockroachdb/cockroach:v1.0 <span class="nb">start</span> --insecure</code></pre></div>
```

Comment:
Thanks for this update, @arjunyel! Instead of manually specifying the version, though, can you please use the `{{site.data.strings.version}}` tag to automatically populate it?

```
cockroachdb/cockroach:{{site.data.strings.version}}
```